src/textual/app.py

```diff
@@ -875,10 +865,15 @@ def display(self, renderable: RenderableType) -> None:
             return
         if not self._closed:
             console = self.console
+            if self._sync_available:
+                console.file.write("\x1bP=1s\x1b\\")
```

Comment:
it could be nice to extract these 2 special ANSI sequences (`\x1bP=1s\x1b\` and `\x1bP=2s\x1b\`) into module-scoped constants, so their name would carry a semantic?
(I'm not even sure what they represent to be honest - are these "Clear To Left" and "Clear All"?)

Reply:
You're right. They should really go in to Rich so I could do `console.print(SyncStart())`

They tell the terminal that an update is about to start / end, so it will try and do it in a single frame. In theory it reduces flickering.